components/etherlink-l2-side/src/ERC20Wrapper.sol

```diff
@@ -2,65 +2,66 @@
 pragma solidity >=0.8.21;
 
 import {ERC20} from "openzeppelin-contracts/token/ERC20/ERC20.sol";
-import {AccessControlEnumerable} from
-    "openzeppelin-contracts/access/AccessControlEnumerable.sol";
-import {Context} from "openzeppelin-contracts/utils/Context.sol";
 import {BridgePrecompile} from "./BridgePrecompile.sol";
 
+function hashToken(string memory ticketer, uint256 identifier)
```

Comment:
It should probably be easier to have both ticketer and identifier in raw bytes form, i.e. 20 bytes per ticketer address + forged identifier (`pair nat (option bytes)`).

That way it would be pretty straightforward to construct tokenHash on the kernel side.

Pros:
- Internal inbox message arrives in raw byte form (same goes for outbox message), we can skip decoding/encoding ticket content/ticketer and manipulate raw bytes instead
- No need to restrict token_id with uint256
- Less ambiguous

Reply:
Sounds good, something like this: [a117b69](https://github.com/baking-bad/ticketer-proto/pull/1/commits/a117b69d9da5b53454cd53a61951ff58d3f982a2), right?

Reply:
Right!